bench/wtperf/wtperf.c

```diff
@@ -382,6 +566,8 @@ worker(void *arg)
 	CONFIG *cfg;
 	CONFIG_THREAD *thread;
 	TRACK *trk;
+	TRUNCATE_QUEUE_ENTRY *truncate_item;
+	TRUNCATE_CONFIG *truncate_config;
```

Comment:
This can be a stack variable. i.e, declare it as:

```
TRUNCATE_CONFIG truncate_config;
```

Then you don't need to malloc it (or free it), and you can pass it as a parameter with ```&truncate_config```